lib/config/validation.ts

```diff
@@ -182,13 +182,7 @@ export async function validateConfig(
               !(val && val.length === 1 && val[0] === '*')
             ) {
               try {
-                RegExp(val as any);
-                if (!safe(val as any)) {
-                  errors.push({
-                    depName: 'Configuration Error',
-                    message: `Unsafe regExp for ${currentPath}: \`${val}\``,
-                  });
-                }
+                regEx(val as any);
```

Comment:
these should have a try/catch and then convert exceptions into `errors` gracefully

Reply:
@rarkins we are already pushing exceptions into errors in catch block. 
errors.push({
                  depName: 'Configuration Error',
                  message: `Invalid regExp for ${currentPath}: \`${val}\``,
                });